### ISSUE TITLE
Project Proposal - Coordinating the Solid Specification

### ISSUE THREAD

**Issue by justinwb** [solid/specification#38]
Proposing that the [Solid Editorial Team](https://github.com/solid/process/blob/master/editors.md#solid-editors) undertakes a project focused on the coordination and orchestration of the v1.0 Solid Specification.

**Goal:**
Coordinate the completion of a comprehensive and reliable v1.0 specification for Solid.

**Approach**
The [Solid Editorial Team](https://github.com/solid/process/blob/master/editors.md#editorial-team), supported by additional subject matter experts, will provide guidance, encouragement, and coordination to any known [panels](https://github.com/solid/process/blob/master/panels.md#solid-panels), individuals, or groups working on Candidate Proposals to the [Solid Specification](https://github.com/solid/specification), helping to orchestrate and optimize those efforts to minimize blocking dependencies and maximize high-quality output that will pass in-depth [editorial review](https://github.com/solid/process#reviewing-proposals).

**Background**
Now that we have an [established process](https://github.com/solid/process), it's time to use it to coordinate our various workstreams into one concerted effort to complete the v1.0 Solid Specification. We can use the structures we now have in place to focus and encourage the work and time people are spending towards this common aim. I believe that an important responsibility of the editorial team is to proactively provide this guidance and coordination. Issues such as solid/process#135, solid/app-authorization-panel#36, and solid/app-authorization-panel#33 underscore the need for this. 

**Proposed Next Steps:**

- **Itemize the work** - Break down the various elements of the specification that are needed for 1.0 as work items.
- **Prioritize the efforts** - Consider the dependencies. In what order should this work be tackled? Which are so tightly intertwined that they need to be tackled together?
- **Identify active groups and workstreams** - Which [panel](https://github.com/solid/process/blob/master/panels.md#solid-panels) is most equipped to undertake a given item of work? Does one exist? Is it in a healthy working state already?
- **Coordinate efforts** - Collaborate with panels to guide and encourage their efforts towards the prioritized 1.0 Specification plan. What kind of help or encouragement might that panel need to produce output that can support this initiative?
- **Plan, Support, and Iterate** - Publish an overarching plan of attack. Identify efforts for each work item by a given panel or individual. Provide regular updates to the community, and work to constantly keep things heading in the right direction at an acceptable cadence.

**Comment by kjetilk** [comment]
Yes, indeed! Lets try to find some time to have a proposal for this together when we meet.

**Comment by RubenVerborgh** [comment]
Fully support this.

Perhaps just replace "Prioritize" by "Determine an order". I want to avoid any perception of it being a judgement of importance.

**Comment by dmitrizagidulin** [comment]
+1 to all this, and @RubenVerborgh's comment about Determining an Order.

**Comment by justinwb** [comment]
> Fully support this.
> 
> Perhaps just replace "Prioritize" by "Determine an order". I want to avoid any perception of it being a judgement of importance.

Adjusted the language above to reflect.

**Comment by csarven** [comment]
s/plan of attack/execution plan? :p 

Sounds good to me.

**Comment by justinwb** [comment]
> s/plan of attack/execution plan? :p
> 
> Sounds good to me.

@csarven Adjusted the language above from _plan of attack_ to _execution plan_  😄

**Comment by michielbdejong** [comment]
One thing I think should definitely in 1.0, even if nothing else does, is attenuated app authorization.

**Comment by RubenVerborgh** [comment]
> attenuated app authorization

Very basic question, but what is it, and do we have pointers to it?

**Comment by michielbdejong** [comment]
Ah sorry, basically it's https://forum.solidproject.org/t/read-only-or-sub-folder-oidc-scopes/767 and https://github.com/solid/authorization-and-access-control-panel - in a sentence, and at the minimal level:

#### If Alice give Bob read-access to (a part of) her pod, then how can Bob give a document-viewer app "attenuated" access to only one resource on there, but not to others.

So the goal is a situation where Alice has control access to her entire pod, Bob has read access to some part of it, and the document-viewer app has read access to only one document, so a strict subset of the access that Bob himself has.

In theory this is already possible: Bob emails Alice the origin of the app and which doc it should be allowed to open, and Alice goes and edits the ACL with an `acl:origin` entry, then emails Bob back. But in practice we (obviously) want a better system for this.

**Comment by RubenVerborgh** [comment]
Thanks, let’s make a specific issue for this. Lots of dependencies there though, so many other things that need a solution first before we can properly tackle it.
